ledfx/devices/artnet.py

```diff
@@ -93,15 +101,19 @@ def config_use(self, config):
         self.data_max = self.num_devices * self.pixels_per_device
 
         self.channel_count = (
-            self.pre_amble.size
-            + (self.pixels_per_device * 3)
-            + self.post_amble.size
-        ) * self.num_devices
+            self.dmx_start_address
+            + (
+                self.pre_amble.size
+                + (self.pixels_per_device * 3)
+                + self.post_amble.size
+            )
+            * self.num_devices
+        )
```

Comment:
Outside brackets here are redundant, you probably get the right answer due to bod math, but recomment change this to be more explicit for future readers

=   self.dmx_start_address + (
            self.num_devices * (
                self.pre_amble.size
                + ( self.pixels_per_device * 3)
                + self.post_amble.size ) )
                
 I reordered the position of the one term as well for readability.

Reply:
I have now introduced a new temp variable to make this term shorter